packages/truffle-contract-schema/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "3.0.11",
   "description": "JSON schema for contract artifacts",
   "main": "index.js",
+  "typings": "./typings/index.d.ts",
```

Comment:
Does this conflict with the types exposed by `truffle-contract-schema/spec`?

Reply:
For whatever reason, we don't currently expose those typings. Should I do so within `typings/index.d.ts`?

Reply:
Yep, I think so!